plugins/splunk-on-call/src/plugin.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createApiFactory,
+  createPlugin,
+  createRouteRef,
+  discoveryApiRef,
+  configApiRef,
+} from '@backstage/core';
+import { splunkOnCallApiRef, SplunkOnCallClient } from './api';
+
+export const rootRouteRef = createRouteRef({
+  path: '/splunk-on-call',
+  title: 'splunk-on-call',
+});
+
+export const plugin = createPlugin({
```

Comment:
Let's stick to https://backstage.io/docs/plugins/composability, as all plugins have now been migrated, https://github.com/backstage/backstage/issues/3424

It means renaming this to `splunkOnCallPlugin`, providing the route ref through `routes` to `createPlugin`, and exporting a page extension component. See for example https://github.com/backstage/backstage/pull/4474/files#diff-c56e8178de33ad86b72f53e6ffcffbb2cbaac07cd6e0c138c6acb86fb5f38165